es/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,566 @@
 
-&#10230;
+&#10230; [Continuo, Clases, Regresión lineal, Regresión logística, SVM, Naive Bayes]
 
 <br>
 
 **7. Type of model ― The different models are summed up in the table below:**
 
-&#10230;
+&#10230; Tipo de modelo ― Los diferentes tipos de modelos se resumen en la siguiente tabla:
 
 <br>
 
 **8. [Discriminative model, Generative model, Goal, What's learned, Illustration, Examples]**
 
-&#10230;
+&#10230; [Modelo discriminatorio, Modelo generativo, Objetivo, Qué se aprende, Ilustración, Ejemplos]
 
 <br>
 
 **9. [Directly estimate P(y|x), Estimate P(x|y) to then deduce P(y|x), Decision boundary,  	Probability distributions of the data, Regressions, SVMs, GDA, Naive Bayes]**
 
-&#10230;
+&#10230; [Estima P(y|x), Estima P(x|y) para luego deducir P(y|x), Límite de decisión,  	Distribución probabilística de los datos, Regresiones, SVMs, GDA, Naive Bayes]
 
 <br>
 
 **10. Notations and general concepts**
 
-&#10230;
+&#10230; Notación y conceptos generales
 
 <br>
 
 **11. Hypothesis ― The hypothesis is noted hθ and is the model that we choose. For a given input data x(i) the model prediction output is hθ(x(i)).**
 
-&#10230;
+&#10230; Hipótesis ― La hipótesis se representa con h0 y es el modelo que elegimos. Para un dato de entrada x(i), la predicción dada por el modelo se representa como h0(x(i)).
 
 <br>
 
 **12. Loss function ― A loss function is a function L:(z,y)∈R×Y⟼L(z,y)∈R that takes as inputs the predicted value z corresponding to the real data value y and outputs how different they are. The common loss functions are summed up in the table below:**
 
-&#10230;
+&#10230; Función de pérdida ― Una función de pérdida es una función L:(z,y)∈R×Y⟼L(z,y)∈R que toma como entrada el valor predicho z y el valor real esperado, dando como resultado qué tan diferentes son ambos. Las funciones de pérdida más comunes son detalladas en la siguiente tabla:
 
 <br>
 
 **13. [Least squared error, Logistic loss, Hinge loss, Cross-entropy]**
 
-&#10230;
+&#10230; [Mínimo error cuadrático, Pérdida logística, Pérdida de bisagra, Entropía cruzada]
 
 <br>
 
 **14. [Linear regression, Logistic regression, SVM, Neural Network]**
 
-&#10230;
+&#10230; [Regresión lineal, Regresión logística, SVM, Red neuronal]
 
 <br>
 
 **15. Cost function ― The cost function J is commonly used to assess the performance of a model, and is defined with the loss function L as follows:**
 
-&#10230;
+&#10230; Función de costo ― La función de costo J es comúnmente utilizada para evaluar el rendimiento de un modelo y se define utilizando la función de pérdida L de la siguiente forma:
 
 <br>
 
 **16. Gradient descent ― By noting α∈R the learning rate, the update rule for gradient descent is expressed with the learning rate and the cost function J as follows:**
 
-&#10230;
+&#10230; Descenso del gradiente ― Siendo α∈R la tasa de aprendizaje, la regla de actualización de descenso del gradiente se expresa junto a la tasa de aprendizaje y la función de costo J de la siguiente manejra:
```

Comment:
* The Google glossary for Machine Learning translates "Gradient descent" as "[_Descenso de gradiente_](https://developers.google.com/machine-learning/glossary/#gradient_descent)".
* You misspelt "_manera_" as "_manejra_".